<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmProgressBarWait</class>
 <widget class="QWidget" name="frmProgressBarWait">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="ProgressBarWait" name="progressBarWait1"/>
   </item>
   <item row="0" column="1">
    <widget class="ProgressBarWait" name="progressBarWait2"/>
   </item>
   <item row="0" column="2">
    <widget class="ProgressBarWait" name="progressBarWait3"/>
   </item>
   <item row="0" column="3">
    <widget class="ProgressBarWait" name="progressBarWait4"/>
   </item>
   <item row="1" column="0">
    <widget class="ProgressBarWait" name="progressBarWait5"/>
   </item>
   <item row="1" column="1">
    <widget class="ProgressBarWait" name="progressBarWait6"/>
   </item>
   <item row="1" column="2">
    <widget class="ProgressBarWait" name="progressBarWait7"/>
   </item>
   <item row="1" column="3">
    <widget class="ProgressBarWait" name="progressBarWait8"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProgressBarWait</class>
   <extends>QWidget</extends>
   <header>progressbarwait.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
